### PR TITLE
validate scripts in container

### PR DESCRIPTION
**Pull request by runcom**
- still missing container in Travis to run test&validate scripts there :cry: 
- still missing docker in the test container which is needed to push/pull from internal registries or docker.io

Signed-off-by: Antonio Murdaca runcom@redhat.com

### CODE REVIEW

File: hack/make.sh [hunk 1]
```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+set -e
+
+# This script builds various binary artifacts from a checkout of the docker
+# source code.
+#
+# Requirements:
+# - The current directory should be a checkout of the docker source code
```
Comment (vdemeester) [inline hack/make.sh]: Maybe we should update that :stuck_out_tongue: (and a little up, and a little down too)

File: hack/make.sh [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	} >&2
+fi
+
+echo
+
+# List of bundles to create when no argument is passed
+# TODO(runcom): these are the one left from Docker...for now
+# test-unit
+# validate-dco
+# validate-pkg
```
Comment (vdemeester) [inline hack/make.sh]: We can safely remove `validate-pkg` and `validate-test` from the comment.. pretty sure we will never need them (really docker specific)

**Comment by vdemeester** [comment]
Few nits, but SGTM :smile_cat:

File: hack/make.sh (hunk 1 above)
Comment (runcom) [inline hack/make.sh]: I'll remove this as a whole lol

File: hack/make.sh (hunk 2 above)
Comment (runcom) [inline hack/make.sh]: cool!

**Comment by runcom** [comment]
cool! I'll merge after Travis is ok with this (hopefully :))

**Comment by vdemeester** [comment]
It's green and good ! :+1:

**Comment by runcom** [comment]
:tada:

**Pull request merged**